client/src/Components/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "../App.css";
+import { BrowserRouter as Link } from "react-router-dom";
+
+
+const Navbar =(props) => {
+	return(
+		<nav className="navbar">
+			<a href="/">
+				<img
+					src="https://codeyourfuture.io/wp-content/uploads/2019/03/cyf_brand.png"
+					alt="cyf_brand.png"
+					className="cyf-log"
+				/>
+			</a>
+			<a href="/mentors/5wjhfxnr" className="link-button">
```

Comment:
Do you think `props.mentors` / `props.results` / `props.newquestion` will be different going forward? It looks like you're calling this component with the same values twice, so I wonder if it makes it a bit cleaner to just hard-code those strings in here rather than relying on `props`. (Feel free to ignore this if you had a specific reason in mind for using `props`.)

Reply:
Hi Shanthi, we did this to be able to use one navbar component that can be reused in our different pages, without a need for doubling of navs from the main page.

Reply:
Yes, re-using the component makes a lot of sense, I'm just wondering about the passing of `props` into it (if the `props` aren't changing across the two calls to the component).

Reply:
Do you mean Sabia can adds a static nav bar for main without links  and than for mentor pages just use nav bar component without props?

Reply:
I meant instead of 
```
<a href="/mentors/5wjhfxnr" className="link-button">{props.mentors}</a>
```

You could just do
```
<a href="/mentors/5wjhfxnr" className="link-button">Mentors</a>
```

but it looks like the code has been updated to include the icon, which we want everywhere, so that's fine. 👍  (Now the `props` makes sense because the student pages won't be passing those in, so we still want the page to work.)

Reply:
Thank you for clarification